### PR TITLE
TitleScreen

### PR DESCRIPTION
**Pull request by Datcat2**
Title screen with a title, exit, and play buttons. There's also an empty window after you press play. In total there's three classes the main, launchpage and newwindow.

### CODE REVIEW

File: TitleScreen/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst
Comment (bvi1994) [inline TitleScreen/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst:1]: You may want to put this file inside a `.gitignore` file. We don't want to track this file as this is specific to your computer and build. Here's what you should do:

In the main level, create a file called `.gitignore`. Then add the following to the `.gitignore` file:

```/target/```

Make sure you save it. Then you would do the following commands:

```git rm -r --cached target```

```git commit -m "Add target directory to .gitignore"```

```git push```

This should remove the file from the PR and we no longer need to track it.

File: TitleScreen/src/main/java/com/mycompany/game/LaunchPage.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  exitButton.setBounds(250,400,200,40);//exit button
+  exitButton.setFocusable(false);
+  exitButton.addActionListener(this);
+  frame.add(exitButton);
+  
+ }
+
+ @Override
+ public void actionPerformed(ActionEvent e) 
+ {
```
Comment (bvi1994) [inline TitleScreen/src/main/java/com/mycompany/game/LaunchPage.java]: I think this is more of a style thing than it is a code thing. You may want to put the braces like this

```
public void actionPerformed(ActionEvent e) {
    [your code]
}

File: TitleScreen/src/main/java/com/mycompany/game/LaunchPage.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  
+  if(e.getSource()==myButton) 
+  {
+   frame.dispose();
```
Comment (bvi1994) [inline TitleScreen/src/main/java/com/mycompany/game/LaunchPage.java:54]: Fix the braces here too

File: TitleScreen/src/main/java/com/mycompany/game/LaunchPage.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+   NewWindow myWindow = new NewWindow();
+  }
+  
+  if (e.getSource() == exitButton)
```
Comment (bvi1994) [inline TitleScreen/src/main/java/com/mycompany/game/LaunchPage.java]: Fix braces and spacing here too in this if statement.

File: TitleScreen/src/main/java/com/mycompany/game/LaunchPage.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+ */
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.event.*;
+import javax.swing.*;
+
+
+public class LaunchPage implements ActionListener{
+ 
+ JFrame frame = new JFrame();
```
Comment (bvi1994) [inline TitleScreen/src/main/java/com/mycompany/game/LaunchPage.java:18]: Keep the spacing consistent. I think you may need to set your editor where it tabs consistently, usually 4 spaces is the norm. Ideally, it should be like this

```
  JFrame frame = new JFrame();
  JButton myButton = new JButton("Play");
  JButton exitButton = new JButton("Exit");

  LaunchPage(){

  frame.setTitle("Snake Game");//Window
  frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
  frame.setSize(700,600);
  frame.setLayout(null);
  frame.setLocationRelativeTo(null);
  frame.setVisible(true);
```

File: TitleScreen/src/main/java/com/mycompany/game/LaunchPage.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+ */
+package com.mycompany.game;
+
+/**
+ *
+ * @author bhlyn
+ */
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.event.*;
```
Comment (bvi1994) [inline TitleScreen/src/main/java/com/mycompany/game/LaunchPage.java:13]: I like how you're only importing what is needed.

File: TitleScreen/src/main/java/com/mycompany/game/NewWindow.java [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+ * @author bhlyn
+ */
+import java.awt.*;
+import javax.swing.*;
+public class NewWindow {
+
+ JFrame frame = new JFrame();
+ JLabel label = new JLabel("Hello!");
+ 
+ NewWindow(){
```
Comment (bvi1994) [inline TitleScreen/src/main/java/com/mycompany/game/NewWindow.java:18]: Fix the spacing in this file too. Remember to keep the spacing consistent.

**Comment by bvi1994** [comment]
The code quality looks good. You do need to add another fix, in particular removing the `TitleScreen/src/main/java/com/mycompany/game/LaunchPage.java` file and adding `*/target/` in `,gitignore` as that would cause errors if we were to develop on it. 

Please address the main issue above before I merge it. Also, it would be great to fix the formatting errors that I mentioned too! :)

**Review by Datcat2 — COMMENTED** [review]
I think I did everything? Took awhile for gitignore to work

**Comment by bvi1994** [comment]
LGTM. Approved and merged.

**Review by bvi1994 — APPROVED** [review]
LGTM. merged

**Pull request merged**